src/components/Footer/Footer.js

```diff
@@ -1,4 +1,171 @@
 import styled from 'styled-components';
+import {
+  faTwitter,
+  faFacebookF,
+  faInstagram,
+  faYoutube,
+} from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
 export default function Footer() {
-  return <div />;
+  return (
+    <Footerwrapper>
```

Comment:
```suggestion
    <FooterWrapper>
```
컴포넌트 이름들을 전부 첫 글자만 대문자로 쓰셨는데 카멜 케이스로 작성하신다면 위와 같이 적으셔야 합니다
전체적으로 네이밍이 틀렸으니 잘못 쓰신 부분 찾아서 수정해주세요

Reply:
스타일드 컴포넌트 관련 네이밍을 카멜케이스로 작성하는법에대해서 잘못알고있었습니다. 리뷰 반영해 수정하도록 하겠습니다!